docs/system-pages/vpn.md

```diff
@@ -0,0 +1,29 @@
+# Налаштування VPN / VPN Setup
```

Comment:
docs in Ukrainian should be with `-ua` suffix

Reply:
It have both Ukrainian and English.
We need to have both on same page, since if you forget to enable VPN, this page will be opened.
It also not supposed to be opened manually.

Reply:
I see. LGTM then.
Maybe @Arriven @iamtodor have concerns

Reply:
LGTM +1